src/main/java/it/cnr/isti/hpc/wikipedia/parser/ArticleParser.java

```diff
@@ -519,15 +521,17 @@ private void setLists(Article article, ParsedPage page) {
 	private void setDisambiguation(Article a) {
 
 		for (String disambiguation : locale.getDisambigutionIdentifiers()) {
-			if (StringUtils.containsIgnoreCase(a.getTitle(), disambiguation)) {
+			if (StringUtils.containsIgnoreCase(a.getTitle(), "(" + disambiguation + ")")) {
```

Comment:
Cool, might be good to add a note somewhere for having a second pass on the logs it produces, so we can double check later.